.gitignore

```diff
@@ -0,0 +1,22 @@
+# Distribution / packaging
+.Python
```

Comment:
Shouldn't `*.pyc` be here?

Reply:
I prefer to store such common ignore in local gitignore. But maybe it's a bad way for opensource

Reply:
Yep, for opensource it's better to have all the relevant things in it.